sdk/spring/azure-spring-cloud-eventhubs-stream-binder/src/main/java/com/azure/eventhub/stream/binder/provisioning/EventHubProducerDestination.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.microsoft.azure.eventhub.stream.binder.provisioning;
+package com.azure.eventhub.stream.binder.provisioning;
```

Comment:
Why is there no 'spring' in the package names for the libraries? I was expecting to see 'com.azure.spring....'

Reply:
We are changing the group id to com.azure.spring in another PR. Since this PR only changes a part of our modules, so we will use another PR to change the group id all together.